lib/widgets/layouts/adaptive_layout/app_adaptive_scaffold_body.dart

```diff
@@ -11,6 +12,7 @@ typedef OnOpenSearchPage = Function();
 typedef OnCloseSearchPage = Function();
 typedef OnClientSelectedSetting = Function(Object object, BuildContext context);
 typedef OnDestinationSelected = Function(int index);
+typedef OnPopInvoked = Function(bool);
```

Comment:
please add the return type also. if not, Dart will make return type as dynamic by default.
As the best practice say: https://dart.dev/tools/linter-rules/always_specify_types

Reply:
done